internal/xds/xdsclient/clientimpl.go

```diff
@@ -222,7 +222,7 @@ func buildXDSClientConfig(config *bootstrap.Config, metricsRecorder estats.Metri
 		Servers:            gServerCfgs,
 		Node:               gNode,
 		TransportBuilder:   grpctransport.NewBuilder(grpcTransportConfigs),
-		ResourceTypes:      supportedResourceTypes(config, gServerCfgMap),
+		ResourceTypes:      supportedResourceTypes(config),
```

Comment:
We still need to pass the map from here, so that it can be passed to the cluster resource type.